test/unit/edition/related_policies_test.rb

```diff
@@ -96,13 +96,12 @@ class Edition::RelatedPoliciesTest < ActiveSupport::TestCase
     assert_equal policy.title, edition.policies[0].title
   end
 
-  [:news_article,
-   :document_collection,
-   :consultation,
-   :publication,
-   :detailed_guide,
-   :speech,
-  ].each do |document_with_policies|
+  %i[news_article
+document_collection
+consultation
+publication
+detailed_guide
+speech].each do |document_with_policies|
```

Comment:
this might be better expressed as:

```ruby
  %i[
    news_article
    #...
    speech
  ].each do |document_with_policites|
    test "....." do
      # whatever
    end
  end
```

I think the single char indents have confused rubocop